client/src/app/management/components/committee-list/committee-list.component.ts

```diff
@@ -59,6 +63,23 @@ export class CommitteeListComponent extends BaseListViewComponent<ViewCommittee>
         this.router.navigate(['create'], { relativeTo: this.route });
     }
 
+    public async forwardToCommittees(): Promise<void> {
+        const content = this.translate.instant(
+            'This will add or remove the following groups for all selected participants:'
```

Comment:
Thats probably an artifact? Not sure what the correct hint would be

`Set or remove forwarding from the selected committees:`

or something